README.md

```diff
@@ -1 +1,200 @@
-# hello-microverse
\ No newline at end of file
+# 📖 # Hello Microverse
+
+**Hello Microverse** is a HTML and CSS project repository with best practices, follow Github flow and a descriptive and easy to understand README file .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```

Comment:
- [ ] Nice way to add the [README](https://github.com/microverseinc/readme-template) , However, kindly add the header as shown in the README template. This was included to give your profile a professional view. 👍 

## Template view 
![image](https://user-images.githubusercontent.com/50990925/226771172-72ba7283-4536-4234-8248-589265181719.png)

## Your view:
![image](https://user-images.githubusercontent.com/50990925/226771228-b179ba45-3863-41a3-a682-7ba69b928f98.png)